optuna/study/study.py

```diff
@@ -174,6 +174,20 @@ def best_trials(self) -> List[FrozenTrial]:
 
         return _get_pareto_front_trials(self)
 
+    @property
+    def cv_results(self) -> Dict[str, Any]:
+        """Return the dictionary has list of all trials.
+
+        Returns:
+            A list of results of all trials.
+        """
+        cv_results_dict_in_list = [trial_.user_attrs for trial_ in self.trials]
+        cv_results_list_in_dict = {
+            key: [dict_[key] for dict_ in cv_results_dict_in_list]
+            for key in cv_results_dict_in_list[0]
+        }
+        return cv_results_list_in_dict
```

Comment:
Could you consider moving this implementation to `optuna/integration/sklearn.py`?
`Trial.user_attrs` can hold a variety of data, not just CV results, so the current implementation might be specific to `OptunaSearchCV`.

Reply:
> your proposal contains typo

Oops! Sorry for bothering you. Please ignore my comment and overwrite it with your change.